source/_patterns/02-molecules/hero/hero-tiles.twig

```diff
@@ -36,6 +36,7 @@
   {% if style == 'background-image' %}
     <div class="jcc-hero-tiles__background">
       <img src="{{ hero_tiles.images.banner }}" alt="Hero image">
+      <div class="jcc-signature-bottom-right">{{ hero_tiles.images.signature_title }}</div>
```

Comment:
Add a null check.  Only show div if hero_tiles.images.signature_title has value.

Reply:
Added null check.

Reply:
I don't see the null check. test this.

{% if hero_tiles.images.signature_title is not null: %}
<div class="jcc-signature-bottom-right">{{ hero_tiles.images.signature_title }}</div>
{% endif %}